wildcard/match_equivalence_test.go

```
@@ -0,0 +1,165 @@
+		start := time.Now()
+		if Match(pattern, name) {
+			t.Fatalf("pattern %d stars + X should not match %q", stars, name)
+		}
+		if d := time.Since(start); d > 50*time.Millisecond {
+			t.Errorf("Match with %d stars took %v, want well under 50ms", stars, d)
+		}
+	}
+	// Interleaved stars are the harder shape.
+	pattern := strings.Repeat("*a", 32) + "X"
+	start := time.Now()
+	Match(pattern, strings.Repeat("a", 128))
```

golangci-lint (2.12.2)

[error] SA4017: Match doesn't have side effects and its return value is ignored

(staticcheck)